src/InteractiveWindow/Editor/InteractiveWindow.cs

```diff
@@ -66,7 +62,6 @@ private enum ReplSpanKind
         private bool _adornmentToMinimize;
 
         private readonly IWpfTextView _textView;
```

Comment:
Are there fields in this class that can be used on any thread? If not, perhaps uiOnly should be an IInteractiveWindow implementation containing all state, and the outer instance simply delegates to the inner instance, after switching to the UI thread.